### ISSUE TITLE
Ausführlicher LOG

### ISSUE THREAD

**Issue by BOSFF** [BOSWatch/BW3-Core#126]
Hallo zusammen.
Ich stoße immer wieder auf das Problem, dass ich den Server nicht starten kann, da ein Plugin nicht gefunden wird. 
Der Log Auszug ist dann plugin not found: mysql
Das bringt mir nur relativ wenig fürs Troubleshooting.

Gibt es irgendwie eine Möglichkeit einen Ausführlichen Log auszugeben?
also wo drin steht, was dem Programm fehlen tut, damit es wie gewollt funktioniert?

Ich wäre über eine schnelle Hilfe sehr erfreut.

**Comment by Schrolli91** [comment]
Das sollte sich mit #153 auch erledigt haben denke ich.
Durch den neuen Installer sollten Dependencies robuster sein.
@KoenigMjr was meinst du?

**Comment by KoenigMjr** [comment]
ja, Dependencies-Fehler sollte es nicht mehr geben, außer der Modul/Plugin-Ersteller vergisst das eintragen in die .txt

**Comment by Schrolli91** [comment]
Dann schließe ich das hiermit erst mal.
Das Grundlegene Problem sollte damit ja behoben sein.
